content/security/for-admins/hardened-desktop/settings-management/admin-console-configure.md

```diff
@@ -0,0 +1,68 @@
+---
+description: How to configure Settings Management for Docker Desktop using the Docker Admin Console
+keywords: admin, controls, rootless, enhanced container isolation
+title: Configure with the Docker Admin Console
+---
+
+{{% restricted title="Early Access" %}}
+
+Settings Management in Docker Admin Console is in Early Access and is available to Docker Business customers only.
+
+{{% /restricted %}}
+
+This page contains information for administrators on how to configure Settings Management with the Docker Admin Console to specify and lock configuration parameters to create a standardized Docker Desktop environment across your Docker company or organization.
+
+## Prerequisites
```

Comment:
Add to Prerequisites - They will also need to have a Verified Domain setup for the policies to take effect on the users under their "managed domains"

See the SSO connections doc for similar patterns (SSO also needs a verified domain setup for SSO to take effect on their managed users)